packages/ndla-scripts/package.json

```diff
@@ -14,17 +15,20 @@
   "keywords": [
     "ndla"
   ],
+  "engines": {
+    "node": ">= 18.0.0"
```

Comment:
Denne burde egentlig være >= 20.6.0 for at `import.meta.resolve` skal fungere, men da er det en viss fare for at vi kommer til å få kjeft i vercel-pipelines som kun støtter node 18.